<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled/>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>KÜTÜPHANE OTOMASYONUNA HOŞGELDİNİZ!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>761</width>
      <height>142</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="uye_islemleri">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/images/user.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kitap_islemleri">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/book.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="oduncalma_islemleri">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/odunc_1.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="oduncteslim_islemleri">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/odunc_2.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Kitap İşlemleri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>311</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Üye İşlemleri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>198</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ödünç Teslim Etme İşlemleri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ödünç Alma İşlemleri</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
